Core/Base/ElasticDB.py

```diff
@@ -0,0 +1,96 @@
+########################################################################
+# $Id: $
+########################################################################
+
+""" ElasticDB is a base class used to connect an Elasticsearch database and manages 
+queries.
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                                       import gLogger, gConfig
+from DIRAC.Core.Utilities.ElasticSearchDB        import ElasticSearchDB
+from DIRAC.ConfigurationSystem.Client.Utilities  import getElasticDBParameters
+
+class ElasticDB( ElasticSearchDB ):
+
+  """
+  .. class:: ElasticDB
+
+  :param str dbHost: the host name of the Elasticsearch database
+  :param str dbPort: The port where the Elasticsearch database is listening
+  :param str clusterName: The name of the cluster.
+  """
+  __dbHost = None
```

Comment:
Why do you use these class attributes if you use object attributes in the constructor ?